common/lib/dependabot/pull_request_creator/github.rb

```diff
@@ -9,7 +9,11 @@ module Dependabot
   class PullRequestCreator
     # rubocop:disable Metrics/ClassLength
     class Github
-      MAX_PR_DESCRIPTION_LENGTH = 65_536 # characters (see #create_pull_request)
```

Comment:
I realize this is technically a slight breaking change to rename this to `PR_DESCRIPTION_MAX_LENGTH`, but I think that's fine for this... although technically part of the public API I highly doubt it's used by anyone outside the scope of this library.